internal/keymagic/keyRingManager_test.go

```diff
@@ -2,14 +2,16 @@ package keymagic
 
 import (
 	"context"
+	"github.com/quan-to/chevron/internal/agent"
 	remote_signer "github.com/quan-to/chevron/internal/config"
-	"github.com/quan-to/chevron/internal/database"
 	"github.com/quan-to/chevron/internal/keybackend"
 	"github.com/quan-to/chevron/internal/tools"
 	"github.com/quan-to/chevron/internal/vaultManager"
+	"github.com/quan-to/chevron/pkg/database/memory"
 	"github.com/quan-to/chevron/pkg/interfaces"
 	"github.com/quan-to/chevron/pkg/models"
 	"github.com/quan-to/chevron/test"
+	"github.com/quan-to/slog"
 	"io/ioutil"
 	"testing"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports)